src/base/bittorrent/session.cpp

```diff
@@ -1534,7 +1534,10 @@ void Session::configureNetworkInterfaces(lt::settings_pack &settingsPack)
                           ? ('[' + Utils::Net::canonicalIPv6Addr(addr).toString() + ']')
                           : addr.toString());
             endpoints << (ip + portString);
-            outgoingInterfaces << ip;
+
+            if (ip != "0.0.0.0" && ip != "[::]") {
```

Comment:
I would use named constants if possible, could you test it?
```c++
if ((addr != QHostAddress::AnyIPv4) && (addr != QHostAddress::AnyIPv6))  // no need for curly brackets
    outgoingInterfaces << ip;
```

Reply:
You'll still need to add parentheses to each comparison and the curly brackets can be omitted (this seems to be preferred by other devs too).

Reply:
It doesn't work.

Reply:
It's working for me.